accounting/rewind.go

```diff
@@ -99,7 +99,7 @@ func AccountAtRound(account models.Account, round uint64, db idb.IndexerDb) (acc
 		MinRound: round + 1,
 		MaxRound: account.Round,
 	}
-	txns, r := db.Transactions(context.Background(), tf)
```

Comment:
This flag should be in the `idb.TransactionFilter` object. We're using a "parameter object" pattern pretty heavily in indexer for these things since there are so many options.

Reply:
That sounds like a much better option - I'll fix some of the changes and move the flag in there